lib/janus/connection.js

```
@@ -107,15 +107,18 @@ JanusConnection.prototype.onTimeout = function(message) {
 };
 
 JanusConnection.prototype.onRemove = function() {
-  this._removeSession();
-  serviceLocator.get('logger').info('Removed ' + this);
+  return this._removeSession().finally(function() {
```

Well, you cannot know from this method what will be done with the exception. Thing is now we are silently ignoring them. I prefer to have unhandled exception then to at least know about it. Agree?